deployment/hasura/metadata/databases/AerieSequencing/tables/public_expansion_run.yaml

```diff
@@ -2,17 +2,24 @@ table:
   name: expansion_run
   schema: public
 object_relationships:
-- name: expansion_set
-  using:
-    foreign_key_constraint_on: expansion_set_id
+  - name: expansion_set
+    using:
+      foreign_key_constraint_on: expansion_set_id
 array_relationships:
-- name: activity_instance_commands
-  using:
-    foreign_key_constraint_on:
-      column: expansion_run_id
-      table:
-        name: activity_instance_commands
-        schema: public
+  - name: activity_instance_commands
+    using:
+      foreign_key_constraint_on:
+        column: expansion_run_id
+        table:
+          name: activity_instance_commands
+          schema: public
+  - name: expanded_sequences
```

Comment:
This name shouldn't be plural, as an object relationship implies a 1-1.

Reply:
Ah! This is actually an array relationship, so in that case should it remain plural?

Reply:
Oh! Yes, this is okay. I must've misread where it was when I made the comment.